src/main/scala/Emulator/ROM.scala

```diff
@@ -1,22 +1,124 @@
 package Emulator
 
-import java.io.InputStream
-import java.nio.file.{Files, Path, Paths}
+
+import scala.language.dynamics
+import scala.scalajs.js
+import js.Dynamic.global
+import org.scalajs.jquery.{JQueryAjaxSettings, JQueryXHR, jQuery}
+
+import scala.scalajs.js.typedarray.{Int8Array,TA2AB}
 
 
 class ROM {
-  var is: InputStream = null
+  var fullRom: Array[Byte] = null;
+  var trainer: Array[Byte] = null;
+  var prgRom: Array[Byte] = null;
+  var chrRom: Array[Byte] = null;
+  var PCPRom: Array[Byte] = null;
+  var PCINSTRom: Array[Byte] = null;
```

Comment:
Okay I accidentaly left these semi-colons there, they're now removed, as for the public var, i thought of using it as is from the other objects, may aswell create getter and use private variables.